CHANGELOG.md

```diff
@@ -46,6 +46,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 * (modules/core/04-channel) [\#1160](https://github.com/cosmos/ibc-go/pull/1160) Improve `uint64 -> string` performance in `Logger`.
 * (modules/core/keeper) [\#1284](https://github.com/cosmos/ibc-go/pull/1284) Add sanity check for the keepers passed into `ibckeeper.NewKeeper`. `ibckeeper.NewKeeper` now panics if any of the keepers passed in is empty.
+* (transfer) [\#1342](https://github.com/cosmos/ibc-go/pull/1342) Emitting Sender address from `fungible_token_packet` events in `OnReceivePacket` and `OnAcknowledgementPacket`.
```

Comment:
weird that mergify hasn't included the commit where this changed to `OnRecvPacket`

```suggestion
* (transfer) [\#1342](https://github.com/cosmos/ibc-go/pull/1342) Emitting Sender address from `fungible_token_packet` events in `OnRecevPacket` and `OnAcknowledgementPacket`.
```

Reply:
That was me when fixing the conflicts... Mergify... is... never... wrong. :)